### DISCUSSION TITLE
Map not showing properly

### DISCUSSION THREAD

**Discussion by luke0x90** [visgl/deck.gl#6823] [Q&A]
When developing a React app, the Deck.gl map does not show. It renders a map as shown:

## Production
![image](https://user-images.githubusercontent.com/44372546/163468251-2f407182-770b-44c9-85ab-8e60576b1a50.png)

It was working perfectly until recently with no changes to the codebase. Thinking mapbox tokens or something?

**Comment by luke0x90** [comment]
```
├── @emotion/react@11.7.1
├── @emotion/styled@11.6.0
├── @material-ui/core@4.12.3
├── @material-ui/icons@4.11.2
├── @mui/icons-material@5.4.2
├── @mui/material@5.4.1
├── @mui/styles@5.4.2
├── @mui/x-data-grid-generator@5.5.1
├── @mui/x-data-grid-pro@5.5.1
├── @mui/x-data-grid@5.5.1
├── @testing-library/jest-dom@5.16.1
├── @testing-library/react@12.1.2
├── @testing-library/user-event@13.5.0
├── chart.js@3.7.1
├── react-chartjs-2@4.0.1
├── react-dom@17.0.2
├── react-map-gl@6.1.18
├── react-markdown@8.0.0
├── react-router-dom@6.2.1
├── react-scripts@5.0.0
├── react@17.0.2
└── web-vitals@2.1.4
```

**Reply by luke0x90** [reply]
Works as required on edge/localhost/dev
![image](https://user-images.githubusercontent.com/44372546/163469900-c3037741-3a75-4430-b0ab-45cb8622568f.png)

Doesnt work edge/public/prod
![image](https://user-images.githubusercontent.com/44372546/163470063-db7e7052-ea68-49e9-aa22-b544ac93c793.png)

doesnt work local or prod on chrome

**Reply by Pessimistress** [reply]
https://github.com/visgl/react-map-gl/issues/1266
